modules/sandboxed-containers-common-terms.adoc

```diff
@@ -0,0 +1,27 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+[id="sandboxed-containers-common-terms_{context}"]
+= {sandboxed-containers-first} common terms
+
+The following terms are used throughout the documentation.
+
+Sandbox:: A sandbox is a controlled environment where programs can run. Within the controlled environment, you can run untested or untrusted programs without risking harm to the host machine or the operating system. A sandbox keeps your workloads isolated from the rest of the system.
++
+In the context of {sandboxed-containers-first}, sandboxing is achieved by running workloads in a different kernel using virtualization, providing enhanced control over the interactions between multiple workloads that run on the same host.
+
+Pod:: A pod is a Kubernetes and {product-title} construct that represents resources where containers can be deployed. Containers run inside of pods, and pods are used to specify resources that can be shared between multiple containers.
++
+In the context of {sandboxed-containers-first}, a pod is implemented as a virtual machine. Several containers can run in the same pod on the same virtual machine.
+
+{sandboxed-containers-operator}:: An Operator is a software component that automates operations, which are actions that a human operator could do on the system.
```

Comment:
Could this be a brief concept module instead of a term?

Reply:
@maxwelldb The Operator itself as a building blocks section. Do you think we should move this content there instead? 
@zanetworker thoughts?

Reply:
@maxwelldb the idea of introducing terms here is to make the user quickly familiar with concepts necessary to understand the product flow by consuming content from the same place.

Reply:
@maxwelldb I think the difference with the example you gave us is the much smaller amount of configuration and tuning we expect in the case of sandboxed containers. At the moment at least, I think if we created a concept module, it would be mostly empty.

@zanetworker Do you think this could change in the future?

Reply:
I haven't seen this succinct of a definition of our term `Operator` before in our docs, but I absolutely love it!